### PR TITLE
FACEITLtd.FACEITClient version 1.30.0

### PR DESCRIPTION
**Pull request by SonexFuzz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15598)

### CODE REVIEW

File: manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FACEITLtd.FACEITClient
```
Comment (OfficialEsco) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.installer.yaml:3]:
```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```

File: manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FACEITLtd.FACEITClient
```
Comment (OfficialEsco) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```

File: manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: FACEITLtd.FACEITClient
```
Comment (OfficialEsco) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.yaml:3]:
```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Since #15568 already got merged you need to change the PackageIdentifier to FACEITLTD.FACEITClient and the FACEITLtd folder to FACEITLTD otherwise there will be issues.

File: manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.yaml (hunk 3 above)
Comment (ItzLevvie) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.yaml:3]:
```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```
You'll also have to update the files to reflect the name change.

File: manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml (hunk 2 above)
Comment (ItzLevvie) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```

File: manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.installer.yaml (hunk 1 above)
Comment (ItzLevvie) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.installer.yaml:3]:
```suggestion
PackageIdentifier: FACEITLTD.FACEITClient
```
There's already a folder named `FACEITLTD` (`manifests/f/FACEITLTD`) in the repository; so update it to match that.

File: manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: FACEITLtd.FACEITClient
+PackageVersion: 1.30.0
+PackageLocale: en-US
+Publisher: FACEIT Ltd.
+PackageName: FACEIT Client
```
Comment (OfficialEsco) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml:7]:
```suggestion
PackageName: FACEIT
```
Wops Missed this one

Comment (ItzLevvie) [inline manifests/f/FACEITLtd/FACEITClient/1.30.0/FACEITLtd.FACEITClient.locale.en-US.yaml:7]:
```suggestion
PackageName: FACEIT
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by OfficialEsco** [comment]
You also need to change the folder path from `manifests/f/FACEITLtd/FACEITClient/1.30.0/` to `manifests/f/FACEITLTD/FACEITClient/1.30.0/`

`Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\f\FACEITLtd\FACEITClient\1.30.0', expected 'manifests\f\FACEITLTD\FACEITClient\1.30.0'`

**Review by OfficialEsco — APPROVED** [review]

**Comment by OfficialEsco** [comment]
I would recommend to use GitHub Desktop the next time btw c:

**Pull request merged**